src/model/Util.js

```diff
@@ -0,0 +1,15 @@
+const auth = require('solid-auth-client');
+
+class UserDetails {
+
+     static async getName(){
+        var session = await auth.currentSession();
+        var username = session.webId;
+        username = username.replace('https://', '');
+        username = username.replace('.solid.community/profile/card#me', '');
```

Comment:
Be wary, this wont work if the user is using inrupt provider.

However, I believe this replace can be replaced by a query using ldflex:
https://solid.github.io/ldflex-playground/#%5B'https%3A%2F%2Fuo263624.solid.community%2Fprofile%2Fcard%23me'%5D.name

Reply:
I simply refactored the one we were using in the home view. Good thing is that now we only have to change it in one place.